.github/workflows/clickable.yml

```diff
@@ -0,0 +1,58 @@
+#  youtube-web - Youtube webapp for Ubuntu Touch
```

Comment:
I'm not going to accept this, the action is overly complex and shouldn't be a part of this PR anyway, please remove

Reply:
removed, i think......will check